### DISCUSSION TITLE
Custom Vue components: how to import external libraries?

### DISCUSSION THREAD

**Discussion by zedee** [area17/twill#878] [Q&A]
Following the new instructions from the changelog of 2.0 version, I wonder how can I add extra Vue libraries (for instance, `vue-multiselect`) to my own Components. Running the `twill:dev` will try to find them into the `vendor/node_modules`, but not to the root `node_modules` on my project.

Any ways to sort this?

**Comment by ifox** [comment]
Yes! Totally fair question. At the moment you won't be able to depend on a custom dependency through npm however you can inject custom JavaScript in the head of your Twill form, through the `extra_js_head` Blade stack, using `@push`. In there you could use the CDN version of the dependency you are looking for, or even a build of multiple dependencies you  install and build in your own project. Then when you are in your custom component those dependencies should be available in the global scope.

**Comment by daun** [comment]
I found an easy fix for this. See pull request #617.
